content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -30,14 +31,14 @@ If you enable IPv4/IPv6 dual-stack networking for your Kubernetes cluster, the c
 Enabling IPv4/IPv6 dual-stack on your Kubernetes cluster provides the following features:
 
    * Dual-stack Pod networking (a single IPv4 and IPv6 address assignment per Pod)
-   * IPv4 and IPv6 enabled Services (each Service must be for a single address family)
+   * IPv4 and IPv6 enabled Services
    * Pod off-cluster egress routing (eg. the Internet) via both IPv4 and IPv6 interfaces
 
 ## Prerequisites
 
 The following prerequisites are needed in order to utilize IPv4/IPv6 dual-stack Kubernetes clusters:
 
-   * Kubernetes 1.16 or later
+   * Kubernetes 1.20 or later (Dual-stack as alpha was re-implemented in 1.20)
```

Comment:
This might confuse some people, we only re implemented the service part.

Reply:
Okay. How do you suggest we communicate that these docs only apply in full to the 1.20 version, and may be incorrect for the first alpha? (As far as I understand it, we do not intend to maintain docs for the first alpha implementation.)

Reply:
My suggestion for this is / was: https://github.com/kubernetes/website/pull/24725#discussion_r516152624

Reply:
We've implemented @sftim's suggestion.